LICENSE

```diff
@@ -11,3 +11,204 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+ Apache License
```

Comment:
Apache License should be centered like the text below. Copy first line including spaces from https://raw.githubusercontent.com/hoodiehq/hoodie/master/LICENSE